### ISSUE TITLE
The --sversion option is broken with python 3.11

### ISSUE THREAD

**Issue by glitsj16** [priv-kweihmann/systemdlint#63]
systemdlint crashes when I try to use the `--sversion` option.

OS: Arch Linux
$ pacman -Q python python-anytree python-systemdunitparser systemd systemdlint
python 3.11.7-1
python-anytree 2.12.1-1
python-systemdunitparser 0.2-1
systemd 255.3-1
systemdlint 1.2.1-1

Without --sversion:

```console
$ systemdlint /usr/lib/systemd/system/cronie.service
/usr/lib/systemd/system/cronie.service:3:warning [ReferencedUnitNotFound] - The Unit 'autofs.service' referenced was not found in filesystem
/usr/lib/systemd/system/cronie.service:1:error [Security.NoUser] - Neither User nor DynamicUser is set
/usr/lib/systemd/system/cronie.service:3:warning [ReferencedUnitNotFound] - The Unit 'ypbind.service' referenced was not found in filesystem
```

With --sversion:

```console
$ systemdlint --sversion 245.4 /usr/lib/systemd/system/cronie.service
Traceback (most recent call last):
  File "/usr/bin/systemdlint", line 33, in <module>
    sys.exit(load_entry_point('systemdlint==1.2.1', 'console_scripts', 'systemdlint')())
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/systemdlint/__main__.py", line 10, in main
    _stash = _parser.GetResults()
             ^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/systemdlint/cls/parser.py", line 116, in GetResults
    self.GlobalValidate()
  File "/usr/lib/python3.11/site-packages/systemdlint/cls/parser.py", line 113, in GlobalValidate
    self.__unititems = s.Run(list(set(self.__unititems)), self.__runargs)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/systemdlint/cls/specials.py", line 508, in Run
    stash += self.__access_rootdir(_sub_stash)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/systemdlint/cls/specials.py", line 239, in __access_rootdir
    not UnitItem(section="Service", key="RootImage").IsValidInVersion(self.__version):
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/systemdlint/cls/unititem.py", line 19, in __init__
    self.UnitName = self.__getUnitName()
                    ^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/systemdlint/cls/unititem.py", line 24, in __getUnitName
    if self.File.endswith(".conf"):
       ^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'endswith'
```

On a side-note, it's not very clear to me if --sversion should support xxx.y version format or just xxx.

**Comment by priv-kweihmann** [comment]
That's different bug and the correct syntax for `--sversion` is `x.xx` - will try to fix that least the bug

**Comment by priv-kweihmann** [comment]
@glitsj16 Should be fixed in latest 1.3.0 release - please test and if fixed please close the ticket

**Comment by glitsj16** [comment]
@priv-kweihmann Tested 1.3.0 and confirming this is fixed. Thanks, also for clearing up the format!
